### ISSUE TITLE
HTML output does not get opened in RStudio 1.3 + R 4.0

### ISSUE THREAD

**Issue by GegznaV** [rstudio/rstudio#7152]
### System details

    RStudio Edition : Desktop 
    RStudio Version : 1.3.1023-1 Preview
    OS Version      : Win 10
    R Version       : 4.0.1

### Steps to reproduce the problem

Run functions (see below) that have to open HTML output in a browser:

```r
browseVignettes("grid")
```
Or
```r
if (!require("exams")) install.packages("exams")
exams::exams2html("tstat")
```

### Describe the problem in detail

Nothing happens. The functions should open HTML output in a browser, but hey do not in RStudio 1.3. When I roll back to RStudio 1.2, this issue does not persist.

### Describe the behavior you expected

In RStudio 1.3 functions should work (give the same results) as they do in RStudio 1.2. 

***
<details>

- [x] I have read the guide to submitting good bug reports at https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports . 
- [x] I have installed the latest version of RStudio and confirmed that the issue still persists.
- [ ] If I am reporting a RStudio crash, I have included a diagnostics report. https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.
</details>

**Comment by kevinushey** [comment]
Note: was able to reproduce this in the v1.4 daily builds as well.

**Comment by jmcphers** [comment]
I think this was just reported on Reddit: https://www.reddit.com/r/RStudio/comments/hdw5ym/how_to_open_the_viewer_panel_in_browser/

**Comment by gustavobrp** [comment]
Hello, I think it is the same problem that I'm facing (and reported on Reddit).

Before updating R to 4.0.1 version, I was using the 3.6.2. Today I installed the 3.6.3 and the problem doesn't occur. I can use the "show in new window" pane normally.

I didn't checked if it was also related to the R Studio version (which I also updated yesterday), but I can say that here the problem is related to the R version. For now I'm using 3.6.3.

**Comment by felixhaass** [comment]
Can confirm this issue with R 4.0.2 and RStudio 1.3.959. 

Running `browseVignettes("grid")` from the R console (with R 4.0.2) works fine, but not from RStudio (where nothing happens). This affects also situations where one wants to open the browser through the "Show in new window" button in the Viewer pane, e.g. after running `leaflet::leaflet()` or `reprex::reprex({print(1)})` which doesn't do anything.

When I change the R version back to 3.5.1 it works again as expected.

**Comment by kevinushey** [comment]
Thanks -- I'm going to take a look at this today.

**Comment by kevinushey** [comment]
Looks like this is still fallout from https://github.com/rstudio/rstudio/commit/8bd458049260650ed0f2c82ca8bdf8a6a5a0a70e.

**Comment by R-Hannibal** [comment]
@GegznaV - thanks so much for reporting this! Adding the verified label as the fix looks successful to me in the latest preview (available [HERE](https://rstudio.com/products/rstudio/download/preview/)). Please let us know if you're seeing something different or have any other issues.

Verified Fixed on: RStudio Version - 1.3.1054, R Version - 4.0.2, Windows 10 - Desktop 

Comparison testing between 1.3.1008/1.3.1054, in the old build `browseVignettes("grid")` does nothing, in the new build it opens the html output in a browser successfully.

**Comment by GegznaV** [comment]
I can also confirm that the issue is solved in RStudio 1.3.1054.
